conans/test/unittests/tools/options/test_options_tools.py

```diff
@@ -0,0 +1,60 @@
+from conan.tools.options import (
```

Comment:
Coverage is already excellent with ``integration`` test, I think this can be removed in favor of the integration tests.

Reply:
I prefer to keep the unit tests, I think they are easier to understand and modify them during development

Reply:
On the contrary, my experience is the opposite. They are a burden to maintain, changing something in internals, a small refactor, etc, results in broken unit tests that requires more time fixing than the refactor itself. While the integration tests are rock solid, and provide a perfect coverage already. Is there any functionality in these unittests that is not covered by the integration tests? If that is the case, then it would make sense to keep them, but if the functionality is already covered by the integration tests, I'd prefer to remove the unit tests. Mocks are not easier to understand when something breaks and need maintenance (and it happens quite often, and it is a pain)